### ISSUE TITLE
Arrow for interstatement connections

### ISSUE THREAD

**Issue by ermannolocascio** [ESI-FAR/INA-tool#73]
just make the connection as arrow. Low priority

**Comment by sjvrijn** [comment]
This depends on #74, otherwise the arrow heads will be hidden underneath the statement node shape

**Comment by sjvrijn** [comment]
Implementation can be done using [`<marker>`](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/marker) tags
